test/fail_compilation/ice17074.d

```diff
@@ -1,19 +1,13 @@
 /*
-TEST_OUTPUT:
```

Comment:
Removing an expected error from the test suite doesn't look like refactoring

Reply:
It's not an expected error. __overloadSet is never used.

Reply:
To put it another way, `__overloadset` is not a D keyword (though the double-underscore means that it is reserved for the compiler to claim or not whenever it wishes).

Reply:
Essentially, this PR frees __overloadset, so technically, I guess it's ok to modify this test.